src/models/Event.ts

```diff
@@ -0,0 +1,8 @@
+import type { DatadropClient } from "../datadrop.js";
+
+export interface Event {
+    name: string;
```

Comment:
The event name could be type-secured to force specific values. That could also help to design an execute function that takes only specific params depending on the event's name.